dali/python/nvidia/dali/eager.py

```diff
@@ -184,7 +184,17 @@ def _prep_kwargs(kwargs, batch_size):
     return kwargs
 
 
+def _desc_call_args(inputs, args):
+    """Returns string description of call arguments (inputs and input arguments) to use as part of
+    the caching key."""
+    return str([(inp.dtype, inp.layout(), inp.shape()) for inp in inputs]) + str(sorted(
+        [(key, value.dtype, value.layout(), value.shape()) for key, value in args.items()]))
```

Comment:
I think we can tone it a bit down with storing just the dimension of given input/argument, shape can be changing.

Reply:
done